app/src/main/java/com/materiiapps/gloom/ui/GloomActivity.kt

```diff
@@ -25,27 +27,31 @@ import com.materiiapps.gloom.ui.screens.auth.LandingScreen
 import com.materiiapps.gloom.ui.screens.root.RootScreen
 import com.materiiapps.gloom.ui.theme.GloomTheme
 import com.materiiapps.gloom.ui.transitions.SlideTransition
+import com.materiiapps.gloom.ui.utils.clearRootNavigation
 import com.materiiapps.gloom.utils.LinkHandler
 import com.materiiapps.gloom.utils.LocalLinkHandler
 import com.materiiapps.gloom.utils.deeplinks.DeepLinkWrapper
 import com.materiiapps.gloom.utils.deeplinks.addAllRoutes
-import kotlinx.coroutines.Dispatchers
+import com.materiiapps.gloom.utils.getOAuthCode
+import com.materiiapps.gloom.utils.isOAuthUri
 import kotlinx.coroutines.launch
+import org.koin.android.ext.android.getKoin
 import org.koin.android.ext.android.inject
 
 class GloomActivity : ComponentActivity() {
 
     private val authRepo: GithubAuthRepository by inject()
+    private val gqlRepo: GraphQLRepository by inject()
     private val auth: AuthManager by inject()
     private val prefs: PreferenceManager by inject()
     private lateinit var navigator: Navigator
 
-    @OptIn(ExperimentalAnimationApi::class)
     override fun onCreate(savedInstanceState: Bundle?) {
         installSplashScreen()
         super.onCreate(savedInstanceState)
 
         WindowCompat.setDecorFitsSystemWindows(window, false)
+        getKoin().declare<Context>(this@GloomActivity)
```

Comment:
whats the practical use for activity as context specifically?
launching a new activity?

Reply:
Yeah its used for custom tabs